packages/nova-schema/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@nova/schema",
+  "version": "0.0.0",
```

Comment:
Will the change file bump this to 1.0.0?

Reply:
Yes, tested it locally via `beachball bump` 🙂